### PR TITLE
Fix- css changes 

### PR DESCRIPTION
**Pull request by karnsweta92**

### CODE REVIEW

File: css/pages/discord.css [hunk 1]
```diff
@@ -1,39 +1,81 @@
...
+
+.get-started {
+  text-align: left;
+  font-weight: 550;
+  
+}
+.discord > *{
+  text-align: left;
+  font-weight: 500;
+  /* margin-left: 2%; */
```
Comment (tejasberi) [inline css/pages/discord.css:21]: please remove commented code

File: css/pages/discord.css [hunk 2]
```diff
@@ -1,39 +1,81 @@
...
+
 }
 
-.discord-info{
+.discord-info {
+  
   text-align: left;
   font-weight: 500;
-  margin-left: 2%;
+  /* margin-left: 2%; */
```
Comment (tejasberi) [inline css/pages/discord.css:34]: please remove commented code

File: css/pages/discord.css [hunk 3]
```diff
@@ -1,39 +1,81 @@
...
-.category-img{
-    display: block;
-    margin-left: 2%;
-}
\ No newline at end of file
+@media only screen and (min-width: 651px) and (max-width: 950px) {
+  .category-img {
+    width: 85%;
+  }
+}
```
Comment (tejasberi) [inline css/pages/discord.css:81]: indentation looks better this time. Are you using any pluggin to manage indents and punctuations?

Comment (karnsweta92) [reply css/pages/discord.css:79]: I have used the prettier code formatter the one you had suggested.

File: css/pages/discord.css (hunk 2 above)
Comment (karnsweta92) [reply css/pages/discord.css]: Removed the commented code

File: css/pages/discord.css (hunk 1 above)
Comment (karnsweta92) [reply css/pages/discord.css]: Removed the commented code https://github.com/Real-Dev-Squad/website-welcome/pull/37/commits/9a070ce154bb58140861e8623e31e5870cae9225

**Pull request merged**